.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,8 +16,8 @@
 
 
 **Please check if the PR fulfills these requirements**
-- [ ] The commit message follows our [guidelines](../DEVELOPERS.md#commits)
```

Comment:
Can you please change this and the other link below to be relative? `../blob/master/DEVELOPERS.md#commits` This way, they work the same in forks.

Reply:
Mmm this doesn't work in every situation, using an absolute url does.

I did the following:
- Edit the template file using github UI
- Update the URL to reflect ur changes
- Preview the file
- Click the link to test if it works and see it won't work.

@Narretz Am I missing something? And what's the benefit of using a relative link to a hardcoded branch ?

Reply:
Ah, okay the relative format only works well if you are in the repository view / file tree. It doesn't work as part of the PR template. Just leave it absolute.

Reply:
I think we can use `/Developers.md`!